ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt

```diff
@@ -93,4 +93,13 @@ interface UtxoLedgerService {
 
 
     // TODO CORE-7327 Add verify(signedTx) verify(ledgerTx)
+
+    @Suspendable
+    fun persistTransaction(signedTransaction: UtxoSignedTransaction)
+
+    @Suspendable
+    fun resolveBackchain(signedTransaction: UtxoSignedTransaction, session: FlowSession)
+
+    @Suspendable
+    fun sendBackchain(session: FlowSession)
```

Comment:
An interesting question whether these need to be on the public API (in the long run) - shouldn't all this happen under the hood when you verify a transaction?
But probably ok for now to get it all moving...

Reply:
Yes, these are 100% temporary and don't see them being needed on the API in the future.